java/flight/flight-sql-jdbc-core/src/test/java/org/apache/arrow/driver/jdbc/ConnectionTest.java

```diff
@@ -102,6 +102,33 @@ public void testUnencryptedConnectionShouldOpenSuccessfullyWhenProvidedValidCred
     }
   }
 
+  /**
+   * Checks if a token is provided it takes precedence over username/pass. In this case,
+   * the connection should fail if a token is passed in.
+   *
+   * @throws SQLException on error.
+   */
+  @Test(expected = SQLException.class)
```

Comment:
nit: could we use assertThrows to make it clearer where we expect to throw (and possibly validate the error message too)?

Reply:
sure thing!